Actions/NewRepoAction.cs

```diff
@@ -35,10 +35,10 @@ public void Execute(JObject request, IEnumerable<ISlackAction> actions)
                 {
                     this.logger.LogInformation($"Sending message to '{action.Channel}'");
                     this.slack.Send(action.Channel,
-                        new SlackMessageModel($"New public repository {fullName}", repoHtmlUrl)
+                        new SlackMessageModel($"New repository {fullName}", repoHtmlUrl)
```

Comment:
Hmm could here be actual information if the repository is public or private?

Reply:
Yeah that information can be there, i add enchantment issue about it.